client/src/App.js

```diff
@@ -27,6 +28,7 @@ const App = () => {
           <Route exact path="/game" component={Game} />
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
+          <Route exact path="/gameSetUp" component={GameSetUp} />
```

Comment:
@sjmarsnc can we use `/new` here pretty please?
```suggestion
          <Route exact path="/new" component={GameSetUp} />
```